pkg/transport/kafka-client/kafka_producer_client.go

```diff
@@ -0,0 +1,125 @@
+package kafkaclient
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/confluentinc/confluent-kafka-go/kafka"
+	"github.com/go-logr/logr"
+	kclient "github.com/open-cluster-management/hub-of-hubs-kafka-transport/kafka-client/kafka-producer"
```

Comment:
alias to kafka instead of kclient, as in other places..

Reply:
It is not possible to do that without changing confluent's import into something else (it is currently kafka) and it feels wrong to give it any other name, since it still provides exports such as kafka.Message, kafka.Event...
"kclient" was changed to "kafkaClient" as basically that is what the kafka repo wraps and provides.